packages/react-scripts/config/webpack.config.js

```diff
@@ -427,9 +433,14 @@ module.exports = function (webpackEnv) {
                   loader: require.resolve('@svgr/webpack'),
                   options: {
                     prettier: false,
-                    svgo: false,
+                    svgo: true,
                     svgoConfig: {
-                      plugins: [{ removeViewBox: false }],
+                      plugins: [
+                        {
+                          name: 'preset-default',
+                          removeViewBox: false,
```

Comment:
Looking at the docs this might not 'just work' and the config looks to be a bit more in an object to override: https://github.com/svg/svgo#default-preset

So possibly needs to be

```
params: {
    overrides: {
        removeViewBox: false
    }
}
```